website/source/docs/auth/aws-ec2.html.md

```diff
@@ -1005,8 +1005,13 @@ in its identity document to match the one specified by this parameter.
 <dl class="api">
   <dt>Description</dt>
   <dd>
-    Creates a role tag on the role. Role tags provide an effective way to restrict the
-    capabilities that are set on the role.
+Creates a role tag on the role. Role tags provide an effective way to restrict
```

Comment:
Can we remove "effective" here? It's best-effort. I'd much prefer to explicitly say that role tags can can help restrict capabilities to a subset of role capabilities, but that it should not be relied upon as a security primitive as in real AWS environments adding and removing role tags are often a widely distributed privilege. And maybe reminding that an instance should never be tied to a role that, in a worst case scenario, provides more capabilities than the instance should ever be allowed to have.